frontend/src/APIClients/UserAPIClient.ts

```diff
@@ -0,0 +1,27 @@
+import AUTHENTICATED_USER_KEY from "../constants/AuthConstants";
+import { getLocalStorageObjProperty } from "../utils/LocalStorageUtils";
+import baseAPIClient from "./BaseAPIClient";
+
```

Comment:
Same comments as the resident API client with type, empty array, and console out